docs/src/main/asciidoc/quickstart.adoc

```diff
@@ -160,3 +160,5 @@ $ curl localhost:8080/env
 A property source called `configserver:<URL of remote repository>/<file name>` contains the `foo` property with a value of `bar`.
 
 NOTE: The URL in the property source name is the git repository, not the config server URL.
+
+WARNING: If you use Spring Cloud Config Client, you will need to set the `spring.config.import` property in order to bind to Config Server. You can read more about it https://docs.spring.io/spring-cloud-config/docs/current/reference/html/#config-data-import[here].
```

Comment:
Remove "will".

Change "https://docs.spring.io/spring-cloud-config/docs/current/reference/html/#config-data-import[here]" to "in the Spring Cloud Config Reference Guide".

(The more specific text inside the link is better for SEO, so that searches get people to the right docs.